ZippyIdSDK/ZippyIdSDK/UI/WizardVC.swift

```diff
@@ -259,4 +283,8 @@ extension WizardVC: NextPhotoStep {
         
         delegate.onCompletedWithError(error: error)
     }
+    
+    func onRetryCallback(vc: ErrorVC, verification: ZippyVerification) {
+        self.apiClient = ApiClient(apiKey: verification.requestToken!, baseUrl: ZippyIdSDK.host)
```

Comment:
Vienreiz man bija dīvains gadījums, kurā api tika atgriezts `"state":"failed"` un `"error":${apraksts}`, bet `"request_token"` neeksistēja, kā it kā nevajadzētu notikt? Tāpēc bija crash. I guess tādos gadījumos man likt `                    self.delegate.onCompletedWithError(error: ZippyError.processingFailed)`?

Reply:
nākamajā komitā vismaz tā izdarīju 🤷🏻‍♀️ dīvaini tho

Reply:
Nu, jā, ja tokens neatnāk, tad retry atļauj un viss lai lietotājam paliek tāpat, tikai tokens tiks no nulles prasīts